### PR TITLE
Organizes frontpage

### PR DESCRIPTION
**Pull request by dynasource**
Containes frontpage improvements:
- before it was a bit unorganized
- it lacked installation instructions
- the important badges are now placed in the title
- links are structured in tables
- uninformative / unnecessary parts are removed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
-Yii PHP Framework Version 2
+Yii PHP Framework Version 2 [![Total Downloads](https://poser.pugx.org/yiisoft/yii2/downloads.png)](https://packagist.org/packages/yiisoft/yii2) [![Latest Stable Version](https://poser.pugx.org/yiisoft/yii2/v/stable.png)](https://packagist.org/packages/yiisoft/yii2)
 ===========================
 
-Thank you for choosing Yii 2 - a modern PHP framework designed for professional Web development.
+Thank you for choosing Yii2 - a modern PHP framework designed for professional Web development.
+Yii2 inherits the main spirit behind Yii for being highly extensible, secure and fast.
+We strive to embrace all the best practices applied in modern Web application development.
```
Comment (samdark) [inline README.md]: I'd remove "all" because it's not true :) We're skipping some for the sake of API...

File: README.md [hunk 2]
```diff
@@ -1,66 +1,66 @@
...
 ------------
 
-The minimum requirement by Yii is that your Web server supports PHP 5.4.
+**Requirements**
 
+Yii2 requires `PHP 5.4`. 
 
-DOCUMENTATION
--------------
+**Composer**
```
Comment (samdark) [inline README.md]: Bold should not be used for headers.

File: README.md [hunk 3]
```diff
@@ -1,66 +1,66 @@
...
 
-Yii 2.0 has a [Definitive Guide](http://www.yiiframework.com/doc-2.0/guide-index.html) and 
-a [Class Reference](http://www.yiiframework.com/doc-2.0/index.html) which cover every detail of Yii.
+With [Composer](https://getcomposer.org/) installed, you can install Yii by running the following commands under a Web-accessible folder:
```
Comment (samdark) [inline README.md]: "If you have Composer, you can install" because of "installed ... install" sounds repetitive.

Comment (samdark) [inline README.md]: There's no need to run it under web accessible folder. In contrary, doing that would result in application files right under webroot which is not what we should encourage.

File: README.md [hunk 4]
```diff
@@ -1,66 +1,66 @@
...
+| ------------- | ------------- | ------------- |
+| API | http://www.yiiframework.com/doc-2.0/index.html [(mirror)](http://stuff.cebe.cc/yii2docs/)| |
+| Guide |  http://www.yiiframework.com/doc-2.0/guide-index.html | http://stuff.cebe.cc/yii2-guide.en.pdf |
 
-### Your participation to Yii 2 development is very welcome!
 
-You may participate in the following ways:
+PARTICIPATION
+------------------
+Your participation to Yii 2 development is very welcome! There are many area's in which you can help us
```
Comment (samdark) [inline README.md]: "participation in"?

Comment (samdark) [inline README.md]: "area's" → "areas"

File: README.md [hunk 5]
```diff
@@ -1,66 +1,66 @@
...
+improve our framework:
```
Comment (samdark) [inline README.md]: our framework → the framework?

File: README.md [hunk 6]
```diff
@@ -1,66 +1,66 @@
...
+
+| Type  | Resource |
```
Comment (samdark) [inline README.md]: Should it be a table? It looks like a list to me. Header and type aren't really helpful.

File: README.md [hunk 7]
```diff
@@ -1,66 +1,66 @@
...
 
+DOCUMENTATION
+-------------
+Yii 2.0 has a Definitive Guide and a Class Reference API which cover every detail of Yii.
 
-HOW TO PARTICIPATE
-------------------
+There is also a PDF version of the Definitive Guide and a Definitive Guide Mirror which is updated every 15 minutes.
+
+| Type  | Online | PDF |
```
Comment (samdark) [inline README.md]: Should it be a table? List looks more suitable here.

**Review by samdark — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1,66 +1,66 @@
...
-
-
-REQUIREMENTS
+INSTALLATION
 ------------
 
-The minimum requirement by Yii is that your Web server supports PHP 5.4.
+**Requirements**
 
+Yii2 requires `PHP 5.4`.
```
Comment (cebe) [inline README.md]: I would also add something like "Yii2 requires at least PHP version 5.4. I also runs fine on newer version like 7.x."
State the minimum requirement but don't sound old/outdated.

File: README.md [hunk 9]
```diff
@@ -1,66 +1,66 @@
...
+Yii 2.0 has a Definitive Guide and a Class Reference API which cover every detail of Yii.
 
-HOW TO PARTICIPATE
-------------------
+There is also a PDF version of the Definitive Guide and a Definitive Guide Mirror which is updated every 15 minutes.
+
+| Type  | Online | PDF |
+| ------------- | ------------- | ------------- |
+| API | http://www.yiiframework.com/doc-2.0/index.html [(mirror)](http://stuff.cebe.cc/yii2docs/)| |
+| Guide |  http://www.yiiframework.com/doc-2.0/guide-index.html | http://stuff.cebe.cc/yii2-guide.en.pdf |
```
Comment (cebe) [inline README.md]: ~~the guide has a mirror too.~~

Comment (cebe) [inline README.md]: actually there is no point in mentioning the mirror anymore. It should go down as soon as the new site is there and yiiframework.com is updated once a day anyway.

File: README.md (hunk 6 above)
Comment (cebe) [inline README.md]: yeah, a two column table does not really make sense. list is better.

**Review by cebe — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (cebe) [inline README.md]: Also for the PDF it is better to link to <http://yiiframework.domain-na.me/doc/download/yii-guide-2.0-en.pdf> which is part of the new website and regenerated more frequently.
We could also link to the translated versions:

- es: http://yiiframework.domain-na.me/doc/download/yii-guide-2.0-es.pdf
- fr: http://yiiframework.domain-na.me/doc/download/yii-guide-2.0-fr.pdf
- pl: http://yiiframework.domain-na.me/doc/download/yii-guide-2.0-pl.pdf
- pt-BR: http://yiiframework.domain-na.me/doc/download/yii-guide-2.0-pt-br.pdf
- ru: http://yiiframework.domain-na.me/doc/download/yii-guide-2.0-ru.pdf

File: README.md (hunk 3 above)
Comment (samdark) [inline README.md]: Then it's wrong there as well.

**Comment by samdark** [comment]
These suggestions were already applied to readme.

**Pull request closed without merging**